aeron-driver/src/main/java/io/aeron/driver/Receiver.java

```diff
@@ -46,7 +47,7 @@ public final class Receiver implements Agent
     private final long reResolutionCheckIntervalNs;
     private long reResolutionDeadlineNs;
     private final DataTransportPoller dataTransportPoller;
-    private final OneToOneConcurrentArrayQueue<Runnable> commandQueue;
+    private final ManyToOneConcurrentLinkedQueue<Runnable> commandQueue;
```

Comment:
I think that we should change the queues between the conductor, receiver, sender back to array queues.  If we are concerned about filling these up, we can check the available capacity before attempting to drain the driverCmdQueue in the conductor.  Keep these as linked queues makes the C implementation far more complex.

Reply:
Restored original queues for sender and receiver.